org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -160,6 +160,14 @@ Parameters:
   scicompIncludeDccCollabGroup:      #JC aws-scicomp-include-dcc-collabs
     Type: String
     Default: '906769aa66-b713e6a6-fe9b-4737-919e-c1d4bc4a8447'
+
+  agoraProdAdminGroup:     #JC aws-agoraprod-admins
+    Type: String
+    Default: '906769aa66-af067ea3-406a-4a69-ab39-251fed985d86'
```

Comment:
Forgot, where do I find this in JC?

Reply:
@xschildw it's in organizations account under SSO service -> groups